packages/react-hooks/src/useBlockTime.ts

```diff
@@ -4,33 +4,45 @@
 import type { ApiPromise } from '@polkadot/api';
 import type { Time } from '@polkadot/util/types';
 
-import BN from 'bn.js';
 import { useMemo } from 'react';
 
-import { BN_ONE, extractTime } from '@polkadot/util';
+import { BN, BN_ONE, BN_THOUSAND, BN_TWO, bnToBn, extractTime } from '@polkadot/util';
 
 import { useTranslation } from './translate';
 import { useApi } from './useApi';
 
 type Result = [number, string, Time];
 
-const DEFAULT_TIME = new BN(6000);
+const DEFAULT_TIME = new BN(6_000);
 
-export function useBlockTime (blocks = BN_ONE, apiOverride?: ApiPromise | null): Result {
+// Some chains incorrectly use these, i.e. it is se to values such as 0 or even 2
```

Comment:
Thank you for fixing Moonbeam;s issue.

FWIW, I don't think our use is incorrect. The field is called `MinimumPeriod` not `ExpectedPeriod` or `HalfOfTheExpectedPeriod`, and the docs also say `Beware that this is different to the *expected* period
		/// that the block production apparatus provides.`